bootstrap/ci/release.py

```
@@ -0,0 +1,420 @@
+        raise click.ClickException(
+            f"Changelog directory not found: {notes_dir}\nExpected docs/changelog/{dir_name}/"
+        )
+
+    try:
+        _load_release_note(dir_name, notes_dir)
+    except ValueError as exc:
+        raise click.ClickException(f"Release note content is invalid: {exc}") from exc
+
+
+def _get_submodule_expected_commit(path: str) -> str:
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)